ibm/data_source_ibm_cis_dns_records.go

```diff
@@ -124,7 +125,7 @@ func dataSourceIBMCISDNSRecordsRead(d *schema.ResourceData, meta interface{}) er
 
 	// session options
 	crn = d.Get(cisID).(string)
-	zoneID = d.Get(cisDomainID).(string)
+	zoneID, _, _ = convertTftoCisTwoVar(d.Get(cisDomainID).(string))
```

Comment:
Why you can not catch error ?

Reply:
sometime user might give only domain_id without crn as domain_id.  If it is so, it might give an error and execution will stop there.  We don't want that.  That's why there is no error check and return.